src/Illuminate/Http/Testing/MimeType.php

```diff
@@ -10,773 +10,989 @@ class MimeType
-        'ics' => 'text/calendar',
-        'css' => 'text/css',
-        'csv' => 'text/csv',
-        'html' => 'text/html',
-        'n3' => 'text/n3',
-        'txt' => 'text/plain',
-        'dsc' => 'text/prs.lines.tag',
-        'rtx' => 'text/richtext',
-        'rtf' => 'text/rtf',
-        'sgml' => 'text/sgml',
-        'tsv' => 'text/tab-separated-values',
-        't' => 'text/troff',
-        'ttl' => 'text/turtle',
-        'uri' => 'text/uri-list',
-        'vcard' => 'text/vcard',
-        'curl' => 'text/vnd.curl',
-        'dcurl' => 'text/vnd.curl.dcurl',
-        'scurl' => 'text/vnd.curl.scurl',
-        'mcurl' => 'text/vnd.curl.mcurl',
-        'sub' => 'text/vnd.dvb.subtitle',
-        'fly' => 'text/vnd.fly',
-        'flx' => 'text/vnd.fmi.flexstor',
-        'gv' => 'text/vnd.graphviz',
-        '3dml' => 'text/vnd.in3d.3dml',
-        'spot' => 'text/vnd.in3d.spot',
-        'jad' => 'text/vnd.sun.j2me.app-descriptor',
-        'wml' => 'text/vnd.wap.wml',
-        'wmls' => 'text/vnd.wap.wmlscript',
-        's' => 'text/x-asm',
-        'c' => 'text/x-c',
-        'f' => 'text/x-fortran',
-        'p' => 'text/x-pascal',
-        'java' => 'text/x-java-source',
-        'opml' => 'text/x-opml',
-        'nfo' => 'text/x-nfo',
-        'etx' => 'text/x-setext',
-        'sfv' => 'text/x-sfv',
-        'uu' => 'text/x-uuencode',
-        'vcs' => 'text/x-vcalendar',
-        'vcf' => 'text/x-vcard',
-        '3gp' => 'video/3gpp',
-        '3g2' => 'video/3gpp2',
-        'h261' => 'video/h261',
-        'h263' => 'video/h263',
-        'h264' => 'video/h264',
-        'jpgv' => 'video/jpeg',
-        'jpm' => 'video/jpm',
-        'mj2' => 'video/mj2',
-        'mp4' => 'video/mp4',
-        'mpeg' => 'video/mpeg',
-        'ogv' => 'video/ogg',
-        'mov' => 'video/quicktime',
-        'qt' => 'video/quicktime',
-        'uvh' => 'video/vnd.dece.hd',
-        'uvm' => 'video/vnd.dece.mobile',
-        'uvp' => 'video/vnd.dece.pd',
-        'uvs' => 'video/vnd.dece.sd',
-        'uvv' => 'video/vnd.dece.video',
-        'dvb' => 'video/vnd.dvb.file',
-        'fvt' => 'video/vnd.fvt',
-        'mxu' => 'video/vnd.mpegurl',
-        'pyv' => 'video/vnd.ms-playready.media.pyv',
-        'uvu' => 'video/vnd.uvvu.mp4',
-        'viv' => 'video/vnd.vivo',
-        'webm' => 'video/webm',
-        'f4v' => 'video/x-f4v',
-        'fli' => 'video/x-fli',
-        'flv' => 'video/x-flv',
-        'm4v' => 'video/x-m4v',
-        'mkv' => 'video/x-matroska',
-        'mng' => 'video/x-mng',
-        'asf' => 'video/x-ms-asf',
-        'vob' => 'video/x-ms-vob',
-        'wm' => 'video/x-ms-wm',
-        'wmv' => 'video/x-ms-wmv',
-        'wmx' => 'video/x-ms-wmx',
-        'wvx' => 'video/x-ms-wvx',
-        'avi' => 'video/x-msvideo',
-        'movie' => 'video/x-sgi-movie',
-        'smv' => 'video/x-smv',
-        'ice' => 'x-conference/x-cooltalk',
+          'ez' => 'application/andrew-inset',
```

Comment:
Why has this file got significant "diff" changes? Can you fix the PR so there are no formatting changes, and only the *actual* changes are here - because it is difficult to see what you've changed in this file?

Reply:
I fixed the indentation

Reply:
Btw the update was done by using the same source that symfony is using